karrot/activities/tests/test_activities_api_filter.py

```diff
@@ -131,8 +131,16 @@ def setUp(self):
                 self.member,
             ], date=self.oneWeekAgo
         )
+
         self.feedback = FeedbackFactory(about=self.activityFeedbackAlreadyGiven, given_by=self.member)
 
+        self.activityFeedbackDismissed = ActivityFactory(
```

Comment:
The consistent (locally), yet inconsistent (globally), casing really bothers me here! Just wanted to say so, not for you to fix, but you can if you want! (even though it's a bit unrelated...).